src/content/doc-surrealdb/reference-guide/observability.mdx

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 6
+sidebar_label: Observability
+title: Observability | Reference guides
+description: In SurrealDB, metrics and traces are available if enabled⁠.
+---
+
+# Observability
+SurrealDB can be monitored by enabling the built in observability.
+
+## Enable observability
+To enable observability the `SURREAL_TELEMETRY_PROVIDER` environment variable has to be set to `otlp`. If set to anything else no observability will be available.
```

Comment:
I see the env vars page is missing this one (and the other telemetry ones), I'll make a PR to add them.

https://surrealdb.com/docs/surrealdb/cli/env